makefiles/Makefile.docs-node

```diff
@@ -59,9 +49,12 @@ next-gen-html:
 	echo "GATSBY_PARSER_USER=${USER}" >> .env.production; \
 	echo "GATSBY_PARSER_BRANCH=${GIT_BRANCH}" >> .env.production; \
 	echo "COMMIT_HASH=${COMMIT_HASH}" >> .env.production; \
+	if [ $(filter-out $@,$(MAKECMDGOALS)) = publish]; then \
+		echo "PATH_PREFIX=${PREFIX}/${GIT_BRANCH}" >> .env.production; \
```

Comment:
This would put master into the prefix for the site, is that what we want for this repository?

Reply:
yes! i checked with @sophstad

Reply:
Sue is right actually, we don't want to include the version because node is an unversioned repository. This is where we need to include the logic we discussed over Slack about determining whether or not the repo is versioned:
> If, in the published-branches.yaml file, if the length of version.active > 1, then it is versioned
If the repo is _not_ versioned, then the branch should be omitted.

Reply:
You are right - I forgot that this docs-node is not versioned! 

Sophie in dms with Sue I've been discussing another option:
Remove all the `.env.production` writing logic from next-gen-html and instead have the server write a `.env.production` file that next-gen-html then copies into snooty. What do you think?

Reply:
Sounds great — much easier to maintain consistency across properties. Good idea!